### PR TITLE
Various fixes/improvements for stakeholder demos

### PR DESCRIPTION
**Pull request by bourque**
This PR contains various bug fixes and minor improvements to the web app in response to stakeholder demos (#658)

### CODE REVIEW

**Comment by bourque** [comment]
@mfixstsci would you be able to review this PR?  No rush.  I figure that since we are mostly done with the presentations, this is ready for merging.

**Review by mfixstsci — COMMENTED** [review]
Everything looks straight forward here. Going to test it all out and then will approve it.

File: jwql/website/apps/jwql/templates/view_image.html [hunk 1]
```diff
@@ -34,18 +34,18 @@ <h3>{{ file_root }}</h3>
 		</form>
 		<br>
 
-		<!-- View Header and proposal information -->
+		<!-- Buttons for viewing header, downloading files -->
     	<p>
-	    	<a id="view_header" class="btn btn-primary mx-2" role="button" href="">View Header</a>
-	    	<a class="btn btn-primary mx-2 disabled" role="button" href="#"> View Proposal {{ file_root[2:7] }} in APT</a>
+	    	<a id="view_header" class="btn btn-primary mx-2" role="button" href="{{ url('jwql:view_header', args=[inst, file_root]) }}">View Header</a>
+            <!-- <a id="download_fits" class="btn btn-primary my-2 mx-2" role="button" href='{{ static("") }}preview_images/{{ file_root[:7] }}/{{ file_root }}' download>Download FITS</a>
```
Comment (mfixstsci) [inline jwql/website/apps/jwql/templates/view_image.html:40]: @bourque Did we want to implement this again or are we going to hold off on the download feature for now?

**Review by mfixstsci — APPROVED** [review]
Most changes are just visual/text with some small feature changes. It all looks good to me and I tested it out locally! Send it.

File: jwql/website/apps/jwql/templates/view_image.html (hunk 1 above)
Comment (bourque) [reply jwql/website/apps/jwql/templates/view_image.html:40]: Yeah I think we should do this in a separate PR.  Re-implementing this is not going to be trivial 😓

**Comment by bourque** [comment]
Thanks @mfixstsci !

**Pull request merged**